s3-get-files.sh

```diff
@@ -1,67 +1,146 @@
 #!/bin/bash
 
+# Set the endpoint URL for AWS CLI operations to handle Digital Ocean space.
+export AWS_ENDPOINT_URL=https://nyc3.digitaloceanspaces.com
```

Comment:
Use `awscli` from now on instead of `s3cmd`, even though it is digital ocean space.